packages/aws-cdk/test/integ/cli/cdk-helpers.ts

```diff
@@ -117,10 +117,7 @@ export async function prepareAppFixture() {
   });
 }
 
-/**
- * Cleanup leftover stacks
- */
-export async function cleanupOldStacks() {
+export async function deleteableStacks(prefix: string) {
```

Comment:
A quick JSDoc explaining what this function does would be helpful (I didn't get it the first time, for example).